components/raftstore/src/store/fsm/apply.rs

```diff
@@ -620,6 +620,12 @@ fn should_write_to_engine(cmd: &RaftCmdRequest) -> bool {
 /// Checks if a write is needed to be issued after handling the command.
 fn should_sync_log(cmd: &RaftCmdRequest) -> bool {
     if cmd.has_admin_request() {
+        if cmd.get_admin_request().get_cmd_type() == AdminCmdType::CompactLog {
```

Comment:
Should we also need to skip sync for admin requests like `ComputeHash` and `VerifyHash`?

Reply:
I did not think about the correctness of `ComputeHash` or `VerifyHash`, but the frequency of them are low. The most frequently admin request is `CompactLog` , which accounts for more than 95%.